frontend/src/app/main/ui/components/forms.cljs

```diff
@@ -412,3 +411,36 @@
                                                  "caution" (:caution item))}
             [:span.text (:text item)]
             [:span.icon {:on-click #(remove-item! item)} i/cross]]])])]))
+
+;; --- Validators
+
+(defn all-spaces?
+  [value]
+  (let [trimmed (str/trim value)]
+    (str/empty? trimmed)))
+
+(def max-length-allowed 250)
+
+(def max-uri-length-allowed 2048)
+
+(defn max-length?
+  [value length]
+  (> (count value) length))
+
+(defn validate-length
+  [field errors-msg length]
```

Comment:
The `length` should be a second argument. This is because the name of the validator is `validate-length` and the two most important parameters should come first, not last.